src/repositories/index.spec.ts

```diff
@@ -60,4 +60,41 @@ describe('BaseTypeORMRepo', () => {
             await connection.close();
         });
     });
+
+    describe('findOne', ()=>{
+
+        it('should throw error when invalid id is passed', async () => {
+            const { base, connection } = await setup()
+            const actual = await base.findOne('');
+            const expected: Error = {
+                name: 'Invalid Id Error',
+                message: 'Id passed is not valid',
+            };
+            console.log(actual);
+            expect(actual).toStrictEqual(expected);
+            await connection.close();
+        });
+
+        it('should sucessfully find the resource for given id', async () => {
```

Comment:
as part of this test we should ensure we also add the resource at first place..
if the above test did not run before this then it would have resulted in error